src/Pages/login.tsx

```diff
@@ -1,23 +1,44 @@
-import { ChangeEvent, useState } from "react";
-import { useNavigate } from "react-router-dom";
+import axios from "axios";
+import { ChangeEvent, FormEvent, useState } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
 
 const Login = () => {
-    const [username, setUserName] = useState<string>('');
+    const [email, setEmail] = useState<string>('');
     const [password, setPassword] = useState<string>('');
+    const [incorrectRes, setIncorrectRes] = useState<boolean>(false);
 
     const navigate = useNavigate();
+    const location = useLocation();
 
+    const logIn = (e: FormEvent<HTMLFormElement>) => {
+        e.preventDefault();    
+        axios.post('http://restapi.adequateshop.com/api/authaccount/login', {
```

Comment:
@ShiranthaKellum, it's better if this URL comes from some configuration file, because sometimes projects can have different environments, for example, dev, QA, and prod, depending on the environment these URLs can be different.